python/tank/authentication/login_dialog.py

```diff
@@ -52,6 +54,18 @@
 THREAD_WAIT_TIMEOUT_MS = 5000
 
 
+def _is_context_desktop():
+    """
+    Indicate if we are in the context of the ShotGrid Desktop.
+
+    When the ShotGrid Desktop is used, we want to disregard the value returned
+    by the call to `get_shotgun_authenticator_support_web_login()` when the
+    target site is using Autodesk Identity.
+    """
+    executable_name = os.path.splitext(os.path.basename(sys.executable))[0].lower()
+    return executable_name == "shotgun"
```

Comment:
@jfboismenu I am wondering if I should add a way to override this behaviour ?

We have already had at least one request from a client to allow the Desktop to connect with the PAT for Identity sites.